src/validations/css/stylelint.js

```diff
@@ -0,0 +1,46 @@
+import Validator from '../Validator';
+
+const errorMap = {
+  'syntaxError/Unclosed block': () => ({
+    reason: 'missing-closing-curly',
+  }),
+
+  'lintRule/declaration-block-trailing-semicolon': () => ({
+    reason: 'missing-semicolon',
+  }),
+};
+
+function isSyntaxError(error) {
+  return error.name === 'CssSyntaxError';
+}
+
+class StyleLintValidator extends Validator {
+  constructor(source) {
+    super(source, 'css', errorMap);
+  }
+
+  _getRawErrors() {
+    return System.import('../linters').then(
+      ({stylelint}) => stylelint(
+        this._source
+      ).then(
+        (result) => (result.messages),
```

Comment:
Tiny nitpick, but the parens around `result.messages` are superfluous—you need parens around an object literal to disambiguate curly brackets for a function body, but otherwise you can just write `=> result.messages` etc.